locale/pt-br/texts.po

```diff
@@ -1637,3 +1667,15 @@ msgstr "Confirma que deseja desregistrar este Nano Contract?"
 #: src/components/NanoContract/UnregisterNanoContractModal.js:44
 msgid "Yes, unregister contract"
 msgstr "Sim, desregistrar"
+
+#~ msgid "Sign this message?"
+#~ msgstr "Assinar essa mensagem?"
+
+#~ msgid ""
+#~ "By clicking approve, you will sign the requested message using the first "
+#~ "address derived from your root key on the m/44'/280'/0'/0/0 derivation "
+#~ "path."
+#~ msgstr ""
+#~ "Ao clicar em aprovar, você assinará a mensagem solicitada usando o "
+#~ "primeiro endereço derivado da sua chave raiz no caminho de derivação "
+#~ "m/44'/280'/0'/0/0."
```

Comment:
I'm not sure why these were commented instead of removed

These translations were for the old sign message modal (refactored in this PR)

I removed them